nx-X11/programs/Xserver/hw/nxagent/Imakefile

```diff
@@ -192,6 +194,7 @@ INCLUDES = \
 	-I../../../../../nxcomp \
 	-I../../../../../nxcompshad \
 	-I../../../../extras/Mesa/include \
+	-I../../../../lib/include/X11 \
```

Comment:
here again: most of the include lines are identical to the other sections. Could probably be simplified.